src/commands/mod.rs

```diff
@@ -24,21 +22,36 @@ pub use subdomain::get_subdomain;
 pub use subdomain::set_subdomain;
 pub use whoami::whoami;
 
+use std::str;
+
+const UNKNOWN_ERR: &str =
+    "An unexpected error occurred, try running the command again with RUSTLOG=info";
+
 /// Run the given command and return its stdout.
 pub fn run(mut command: Command, command_name: &str) -> Result<(), failure::Error> {
-    info!("Running {:?}", command);
+    log::info!("Running {:?}", command);
 
-    let status = command.status()?;
+    let output = command.output()?;
+    dbg!(output.clone());
```

Comment:
was this intended to be left in here?

(Also, if this was for debugging purposes,

    let output = dbg!(command.output()?);

would be better)